cypress/integration/service_page_spec.ts

```diff
@@ -0,0 +1,19 @@
+/// <reference types="cypress" />
+/* eslint-disable newline-per-chained-call */
+
+describe('Service Page', () => {
+  const serviceId = 1;
+
+  it('should load a basic page', async () => {
+    cy.request(`/api/services/${serviceId}`).should(r => {
+      expect(r.status).to.eq(200);
+      expect(r.body.service).to.include.all.keys('id', 'name', 'long_description');
+
+      cy.viewport(1200, 1800);
```

Comment:
Is there a way to set this as the default viewport for all tests?

Reply:
This is actually a hangover from me experimenting with taking screenshots at different levels of responsiveness and seeing if the github build artifact API would allow us to see them anywhere. Would be particularly cool if for each build we had visual diffs at various screen sizes, and each deploy our manual QA could involve checking these images for regressions.

Will remove, for these tests the default viewport size is OK.